Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionDataRecord.mm

```diff
@@ -144,6 +146,12 @@ - (unsigned long long)sizeOfDataTypes:(NSSet<_WKWebExtensionDataType> *)dataType
     return createNSArray(records).get();
 }
 
+NSError *createDataRecordError(_WKWebExtensionDataRecordError error, NSString *underlyingErrorMessage)
+{
+    NSDictionary *userInfo = underlyingErrorMessage ? @{ NSUnderlyingErrorKey: underlyingErrorMessage } : @{ };
```

Comment:
`NSUnderlyingErrorKey` is for another `NSError`, not a `NSString`. You can use `NSDebugDescriptionErrorKey` for this. (You should rename the parameter to `debugDescription`.)